Client/game_sa/CPoolsSA.h

```diff
@@ -241,7 +241,7 @@ class CPoolsSA : public CPools
     void ResetPedPoolCount() { m_pedPool.ulCount = 0; }
     void InvalidateLocalPlayerClientEntity();
 
-    uint AddTextureDictonarySlot();
+    uint AddTextureDictonarySlot(std::string strTxdName);
```

Comment:
perhaps use `const std::string&`, because you won't be `std::move`ing the value, so a const ref is fine

Reply:
Thanks